test/transmuxer.test.js

```diff
@@ -2,63 +2,57 @@
 
 var mp2t = require('../lib/m2ts'),
     codecs = require('../lib/codecs'),
-    fs = require('fs'),
     aac = require('../lib/aac'),
     flv = require('../lib/flv'),
     id3Generator = require('./utils/id3-generator'),
     mp4 = require('../lib/mp4'),
     QUnit = require('qunit'),
     testSegment = require('./utils/test-segment'),
     mp4AudioProperties = require('../lib/mp4/transmuxer').AUDIO_PROPERTIES,
-    mp4VideoProperties = require('../lib/mp4/transmuxer').VIDEO_PROPERTIES;
-
-    mp4.tools = require('../lib/tools/mp4-inspector');
-
-var
-
-  TransportPacketStream = mp2t.TransportPacketStream,
-  transportPacketStream,
-  TransportParseStream = mp2t.TransportParseStream,
-  transportParseStream,
-  ElementaryStream = mp2t.ElementaryStream,
-  elementaryStream,
-  AacStream = aac,
-  aacStream,
-  H264Stream = codecs.h264.H264Stream,
-  h264Stream,
-
-  VideoSegmentStream = mp4.VideoSegmentStream,
-  videoSegmentStream,
-  AudioSegmentStream = mp4.AudioSegmentStream,
-  audioSegmentStream,
-
-  AdtsStream = codecs.adts,
-  adtsStream,
-  Transmuxer = mp4.Transmuxer,
-  FlvTransmuxer = flv.Transmuxer,
-  transmuxer,
-  NalByteStream = codecs.h264.NalByteStream,
-  nalByteStream,
-
-  MP2T_PACKET_LENGTH = mp2t.MP2T_PACKET_LENGTH,
-  H264_STREAM_TYPE = mp2t.H264_STREAM_TYPE,
-  ADTS_STREAM_TYPE = mp2t.ADTS_STREAM_TYPE,
-  METADATA_STREAM_TYPE = mp2t.METADATA_STREAM_TYPE,
-  packetize,
-
-  PAT,
-  generatePMT,
-  PMT,
-  standalonePes,
-  validateTrack,
-  validateTrackFragment,
-
-  pesHeader,
-  transportPacket,
-  videoPes,
-  adtsFrame,
-  audioPes,
-  timedMetadataPes;
+    mp4VideoProperties = require('../lib/mp4/transmuxer').VIDEO_PROPERTIES,
+    TransportPacketStream = mp2t.TransportPacketStream,
+    transportPacketStream,
+    TransportParseStream = mp2t.TransportParseStream,
+    transportParseStream,
+    ElementaryStream = mp2t.ElementaryStream,
+    elementaryStream,
+    AacStream = aac,
+    H264Stream = codecs.h264.H264Stream,
+    h264Stream,
+
+    VideoSegmentStream = mp4.VideoSegmentStream,
+    videoSegmentStream,
+    AudioSegmentStream = mp4.AudioSegmentStream,
+    audioSegmentStream,
+
+    AdtsStream = codecs.adts,
+    adtsStream,
+    Transmuxer = mp4.Transmuxer,
+    FlvTransmuxer = flv.Transmuxer,
+    transmuxer,
+    NalByteStream = codecs.h264.NalByteStream,
+    nalByteStream,
+
+    MP2T_PACKET_LENGTH = mp2t.MP2T_PACKET_LENGTH,
+    H264_STREAM_TYPE = mp2t.H264_STREAM_TYPE,
+    ADTS_STREAM_TYPE = mp2t.ADTS_STREAM_TYPE,
+    METADATA_STREAM_TYPE = mp2t.METADATA_STREAM_TYPE,
+    packetize,
+
+    PAT,
+    generatePMT,
+    PMT,
+    validateTrack,
+    validateTrackFragment,
+
+    pesHeader,
+    transportPacket,
+    videoPes,
+    adtsFrame,
+    audioPes,
+    timedMetadataPes;
+
+mp4.tools = require('../lib/tools/mp4-inspector');
```

Comment:
It's an assignment so I have to perform it outside the `var` declarations. We *really* should clean up the declarations in this file sometime, though.